AK/Checked.h

```diff
@@ -1,28 +1,8 @@
 /*
  * Copyright (C) 2011-2019 Apple Inc. All rights reserved.
```

Comment:
Can this really be replaced?

Reply:
No I'll undo that thanks for checking... I found a few of these manually.. but hard to automate.

Reply:
Ok fixed, I manually poked through the diff in vim ... I think this is the only case of this.

Reply:
yep, I just pulled your branch and did a quick `git show | grep -F 'Copyright (c)' | sed -E 's@[0-9-]+,?@@' | sort -u | less` (72 results). everything looks good.

Reply:
Thank you for double checking!